io.itch.domestique-baston.yml

```diff
@@ -0,0 +1,19 @@
+app-id: io.itch.domestique-baston
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: domestique-baston
+modules:
+  - name: domestique-baston
+    buildsystem: simple
+    build-commands:
+      - install -D 'domestique-baston-gles3' /app/bin/domestique-baston
+    sources:
+      - type: file
+        path: 'domestique-baston-gles3'
```

Comment:
this file doesn't exist. and is should be a `url` because upstream build artifacts don't belong here anyway